examples/interaction/tooltips/tooltip_content.py

```diff
@@ -1,3 +1,12 @@
+''' A visualization of adding tooltips to widgets in bokeh.models.
+This example demonstrates defining two distinct tooltip widgets,
+i.e. plaintext and html tooltip, and accepting text input through those widgets.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.io.show, bokeh.models.TextInput, Bokeh.models.Tooltip
+    :refs: ref:`ug_interaction_tooltips
```

Comment:
missing trailing backtick
```suggestion
    :refs: ref:`ug_interaction_tooltips`
```